package.json

```diff
@@ -105,4 +120,4 @@
         "typescript": "^2.9.2",
         "vscode": "^1.1.36"
     }
-}
+}
```

Comment:
single blank line at the end of very file.

Reply:
added back the blank line and now its good, this is the default behavior of vscode formatter